### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by the-faizmohammad**
This pull request introduces several additions to the project. The following sections have been added:

Build Only These 2 Pages:

Home Page: Created a responsive version for two different screen sizes.
About Page: Developed a responsive version for two different screen sizes.
Page Navigation:

Home Page Link: Added a link to the About page in the menu.
Logo Link: Enabled the logo in the header to redirect to the Home page.
Mobile Menu:

Close Button: Implemented a close (X) button that allows users to close the mobile menu.
Dynamic Section:

Featured Speakers: Utilized JavaScript to dynamically generate the "Featured speakers" section.
Project Description: Included a detailed description of the project.

Additionally, the project has been successfully deployed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,175 @@
...
+- **Display colorfull text**
+- **Engaging and interactive design**
+- **Detailed information about the conference agenda and sessions**
+- **Profiles of renowned speakers and thought leaders**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo link](https://https://the-faizmohammad.github.io/conferencePage/)
```
Comment (Ibizugbe) [inline README.md]: The link to your live demo is currently broken.
image

Kindly deploy your project using GitHub pages and update the live demo with the link to the project as this is a requirement in this milestone😉.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                <img class="logo" src="assets/ailogo.png" alt="AI-logo">
+                <p class="mob-menu-para">Join <strong>India's </strong> vision to support <strong>United Nations</strong> in the field of artificial intelligence</p>
+                <a href="#" class="close-link">x</a>
+                <a class="bottom-nav-link" href="about.html"><strong>About</strong></a>
+                <a class="bottom-nav-link" href="#main"><strong>Program</strong></a>
+                <a class="bottom-nav-link" href="#speakers"><strong>Sponsor</strong></a>
+                <a class="bottom-nav-link enroll" href="#"><strong>Buy Ticket</strong></a>
+              </div>
+              <img class="nav-bar-icon" src="assets/hamburger .png" alt="hamburger-icon">
+            </nav>
```
Comment (Ibizugbe) [inline index.html]: Nice job wrapping your navigation links in a `nav` tag and your `nav` tag in a `header` tag as this is a good semantic practice😉.  But let's try to stick to the design by ensuring the Nav is transparent on mobile😉. Kindly take away the white background on the `nav` and make it match the design😉.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

File: index.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+          <header class="mobile-head">
+            <nav class="mobile-nav">
+              <div class="mob-menu-links">
+                <img class="logo" src="assets/ailogo.png" alt="AI-logo">
+                <p class="mob-menu-para">Join <strong>India's </strong> vision to support <strong>United Nations</strong> in the field of artificial intelligence</p>
+                <a href="#" class="close-link">x</a>
+                <a class="bottom-nav-link" href="about.html"><strong>About</strong></a>
+                <a class="bottom-nav-link" href="#main"><strong>Program</strong></a>
+                <a class="bottom-nav-link" href="#speakers"><strong>Sponsor</strong></a>
+                <a class="bottom-nav-link enroll" href="#"><strong>Buy Ticket</strong></a>
```
Comment (Ibizugbe) [inline index.html:42]: The `Buy Tickets` link is not properly aligned to the center to match the design, I suggest you kindly fix that as we are required to stick to the design even as we personalize the project👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

File: index.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+          <div class="bottom-nav-bar">
+            <a href="index.html"><img class="nav-bar-logo" src="assets/ailogo.png" alt="AI-logo"></a>
+            <div class="nav-bar-links">
+              <a class="bottom-nav-link" href="about.html">About</a>
+              <a class="bottom-nav-link" href="#partner">Partner</a>
+              <a class="bottom-nav-link" href="#">Join</a>
+              <a class="bottom-nav-link" href="#">Sponsor</a>
+              <a class="bottom-nav-link" id="ai" href="#">BUY TICKETS</a>
+            </div>
+          </div>
```
Comment (Ibizugbe) [inline index.html:32]: Your navigation bar on the `home` page is blurred as a result of the kind of background image you used for the header😉. and this doesn't match the design. Kindly ensure your nav bar is built to match the given design👍.

yours
image

design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+            global development priorities.
+          </p>
+          <h2 class="header-date">
+            <strong>24 JULY 2023, 17:00-18:30(CEST GENEVA)</strong>
+          </h2>
+          <p class="header-main-subdescription">
+            Join <strong>India's </strong> vision to support united nations in the field of artificial intelligence
+          </p>
+        </div>
+      </div>
```
Comment (Ibizugbe) [inline index.html]: To make your page more semantic, Kindly take these welcome texts away from the header section😉. I suggest the header should only contain the nav and elements relating to navigations😉.  You can probably create an `intro` section with a section tag after the header and put these contents inside it👍.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    </header>
+  </div>
+  <!-- finish header section -->
+  <section class="program" id="program">
+    <h3 class="program-title"><strong>Main Program</strong></h3>
```
Comment (Ibizugbe) [inline index.html]: Nice job so far obeying semantic rules (there is only one `h1` on each page😉). To stay consistent with the header hierarchy rule, let's ensure every other section starts with an `h2` and the subsequent header can follow the next hierarchy😉. Kindly implement this all round your project👍.

Comment (Ibizugbe) [inline index.html]: To match the design, Let's try to implement this underline style in every header where it is required in our project as shown in the design😉.

image

I can see you implemented it in the partner section of your index page, kindly implement it in other required places😉.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+       <div class="our-partner-logo">
+       <img src="assets/py.png" alt="partner 1">
+       <img src="assets/unesco.png" alt="partner 2">
+       <img src="assets/bill.png" alt="partner 3">
+       <img src="assets/unicef.png" alt="partner 4">
+       <img src="assets/un.png" alt="partner 5">
+       <img src="assets/imf.png" alt="partner 6">  
+     </div>
+    </div>
+  </section>
```
Comment (Ibizugbe) [inline index.html]: Kindly ensure the right spacings are added around the images to match the design as this is a requirement in this milestone😉.

design
image

yours
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+  <!--Footer section starts-->
+  <div class="footer" id="footer">
+    <div class="footer-section">
+        <img class="footer-image" src="assets/footer image.png" alt=" footer logo ">
+        <p class="footer-description">Through AI for Good, we have the opportunity to revolutionize fields, empower communities, and find innovative solutions to complex global challenges, ultimately contributing to a brighter future for humanity.
+        </br></br>
+        <strong>AIFG 2023 © All rights reserved.</strong>
+        </p>
+    </div>
+</div>
```
Comment (Ibizugbe) [inline index.html]: Both for your index and about page, I suggest you kindly wrap the footer section in a `footer` tag as this is more semantic😉. also, to match the design, kindly do well to add the necessary spacings (margin and padding)

design
image

Yours
image

Kindly implement these changes in both your index and about page👍.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+     <div class="our-partner-underline" ></div>
+    <div class="our-partner-part">
+       <div class="our-partner-logo">
+       <img src="assets/py.png" alt="partner 1">
+       <img src="assets/unesco.png" alt="partner 2">
+       <img src="assets/bill.png" alt="partner 3">
+       <img src="assets/unicef.png" alt="partner 4">
+       <img src="assets/un.png" alt="partner 5">
+       <img src="assets/imf.png" alt="partner 6">  
+     </div>
```
Comment (Ibizugbe) [inline index.html]: On mobile, your partner images should be properly aligned to match the given design as it is a requirement in this milestone that we stick to the given design😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @the-faizmohammad 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Project has a `home` and `about` page and there's a link to navigate between both pages👍.
- [x] Both pages contain the same nave bar and footer👍.
- [x] On mobile it has a hamburger menu on the left👍.
- [x] The page uses the right color and font family and stated in the design👍.
- [x] All images have an `alt` value👍.
- [x] The logo has a hyperlink to the home page👍.
- [x] The speaker section is generated dynamically👍.
- [x] Nice job implementing the functionalities of the more button👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Try to ensure your commit messages are always capitalized as this is more professional 😉.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+          <p class="header-main-subdescription">
+            Join <strong>India's </strong> vision to support united nations in the field of artificial intelligence
+          </p>
+        </div>
+      </div>
+    </section>
+  </div>
+  <!-- finish header section -->
+  <section class="program" id="program">
+    <h2 class="program-title"><strong>Main Program</strong></h2>
```
Comment (LionRouge1) [inline index.html:74]: - [OPTIONAL] As mentioned by the previous reviewer. Kindly add the red line under the titles to match the Figma design.
251657352-75451004-95df-4b37-8b85-ed02a81c5320

**Review by LionRouge1 — APPROVED** [review]
Hi @the-faizmohammad, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**